chapters/research-design.tex

```diff
@@ -1,288 +1,586 @@
-are sufficient to estimate treatment effects
-given somewhat weaker assumptions on the structure of that effect.
-One solution, as with the experimental variant of 2SLS proposed above,
-is to incorporate matching models into explicitly experimental designs.
-
+are designed to remove some of the dependence on linearity.
+In all ex-post cases, pre-specification of the exact matching model
+can prevent some of the potential criticisms on this front,
+but ex-post matching in general is not regarded as a strong approach.
+
+Analysis of data from matching designs is relatively straightforward;
+the simplest design only requires controls (indicator variables) for each group
+or, in the case of propensity scoring and similar approaches,
+weighting the data appropriately in order to balance the analytical samples on the selected variables.
+The \texttt{teffects} suite in Stata provides a wide variety
+of estimators and analytical tools for various designs.\sidenote{
+  \url{https://ssc.wisc.edu/sscc/pubs/stata_psmatch.htm}}
+The coarsened exact matching (\texttt{cem}) package applies the nonparametric approach.\sidenote{
+  \url{https://gking.harvard.edu/files/gking/files/cem-stata.pdf}}
+DIME's \texttt{iematch} command in the \texttt{ietoolkit} package produces matchings based on a single continuous matching variable.\sidenote{
+  \url{https://dimewiki.worldbank.org/wiki/iematch}}
+In any of these cases, detailed reporting of the matching model is required,
+including the resulting effective weights of observations,
+since in some cases the lack of overlapping supports for treatment and control
+mean that a large number of observations will be weighted near zero
+and the estimated effect will be generated based on a subset of the data.
+
+%-----------------------------------------------------------------------------------------------
 \subsection{Synthetic controls}
 
-\textbf{Synthetic controls} methods\cite{abadie2015comparative}
-\index{synthetic controls}
-are designed for a particularly interesting situation:
-one where useful controls for an intervention simply do not exist.
-Canonical examples are policy changes at state or national levels,
-since at that scope there are no other units quite like
-the one that was affected by the policy change
-(much less sufficient \textit{N} for a regression estimation).\cite{gobillon2016regional}
-In this method, \textbf{time series data} is almost always required,
-and the control comparison is contructed by creating
-a linear combination of other units such that pre-treatment outcomes
-for the treated unit are best approximated by that specific combination.
+\textbf{Synthetic control} is a relatively newer method
+for the case when appropriate counterfactual individuals
+do not exist in reality and there are very few (often only one) treatment unit.\cite{abadie2015comparative}
+  \index{synthetic controls}
+For example, state- or national-level policy changes
```

Comment:
i think of state/national level policy changes as classic examples of natural experiments!

Transport and other big infrastructure projects very commonly use synthetic controls, since few units and placement cannot be randomized

Reply:
Agreed? That's why they fall here. Maybe it should be clearer that it is for the _effect on the state as a single unit_ when the effects at lower levels can't be measured? But this is hard to express...

Reply:
56941c406c46660c75cfd8a4ca7d3bc3423dae52